### ISSUE TITLE
Crash on start

### ISSUE THREAD

**Issue by TheeKeyboardWarrior** [Dimencia/LuteBot3#26]
hello, i am getting a crash on startup along the lines of:
2/5/2023 7:54:36 PM - Crash: A device ID has been used that is out of range for your system.
   at Sanford.Multimedia.Midi.OutputDevice..ctor(Int32 deviceID)
   at LuteBot.Core.Midi.MidiPlayer..ctor(TrackSelectionManager trackSelectionManager)
   at LuteBot.LuteBotForm..ctor()
   at LuteBot.Program.Main()
In method .ctor
In class OutputDevice

2/5/2023 7:54:48 PM - Crash: A device ID has been used that is out of range for your system.
   at Sanford.Multimedia.Midi.OutputDevice..ctor(Int32 deviceID)
   at LuteBot.Core.Midi.MidiPlayer..ctor(TrackSelectionManager trackSelectionManager)
   at LuteBot.LuteBotForm..ctor()
   at LuteBot.Program.Main()
In method .ctor
In class OutputDevice
cant figure out why, can anyone help?

**Comment by TheeKeyboardWarrior** [comment]
1 week later and I'm still having this issue with no response so let me shed some more light on the issue. I have been able to use Lutebot with no issues for nearly 2 years now and I am only just now having this issue, granted I did attempt to start it while playing Rust as I do often preform in that game, not nearly as much as Mordhau though. it might have been a windows update or something with my audio drivers I'm not really sure but it would be a boon if someone, hopefully Dimencia since you probably have the best take on the issue, to assist in this endeavor.
